examples/ads.amp.html

```diff
@@ -747,9 +747,10 @@ <h2>CA A.J.A. Infeed</h2>
   </amp-ad>
 
   <h2>CA ProFit-X</h2>
-  <amp-ad width="320" height="50"
+  <amp-ad width="320" height="100"
```

Comment:
One last request. Can we keep the deprecated usage example here, and add a comment describe your deprecation plan. This is to ensure this is not a breaking change with manual testing. Thank you.

Reply:
@zhouyx Sure, I have updated at https://github.com/ampproject/amphtml/pull/21284/commits/f08528b4dc9e6f51b5b3c69e570f468c32d60972
Please take another look again.